docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -155,9 +155,50 @@ Initially, the status of persistent volume claims might be "Pending"
 but it should become "Bound" fairly quickly.
 
 
+Step 5: Create the Config Map - Optional
+----------------------------------------
+
+This step is only required if you are planning to set up a cross datacenter
```

Comment:
'Cross datacenter' needs DNS names. Within the datacenter/AZ, the names should resolve. But there are gotchas wrt to cloud providers. I will reword it to not include cross datacenter. Nice catch!